src/App.tsx

```diff
@@ -21,6 +31,11 @@ export const App: React.FC = () => {
             data-cy="perPageSelector"
             id="perPageSelector"
             className="form-control"
+            onChange={(e) => {
+              setItemsOnDesc(+e.target.value);
+              setCurrentPage(startValue);
+            }}
```

Comment:
Move to separate function  (transfer the function to a variable, and here only write the name of the function)